OWNERS_ALIASES

```diff
@@ -24,3 +24,8 @@ aliases:
     - soltysh
     - sttts
     - ymqytw
+  test-infra-maintainers:
```

Comment:
Not yet, there are plans to use a different back-end (use github rather than this file) but of course the code needs to be written and we want the feature in first. Feel free to submit a PR to add sen (or modify this pull-request, it should be editable).